calico_containers/pycalico/block.py

```diff
@@ -67,14 +68,24 @@ def __init__(self, cidr_prefix, host_affinity):
         self.cidr = cidr_prefix
         self.db_result = None
 
-        self.host_affinity = host_affinity
+        self.host_affinity = host_affinity or None
```

Comment:
`host_affinity` is a required argument, no?  If the user of this class wants to disable host affinity, they need to explicitly pass `None`, so I think we don't need this "or" clause here.